cmd/package-server-manager/main.go

```diff
@@ -71,6 +67,9 @@ func run(cmd *cobra.Command, args []string) error {
 		LeaderElection:          !disableLeaderElection,
 		LeaderElectionNamespace: namespace,
 		LeaderElectionID:        leaderElectionConfigmapName,
+		RetryPeriod:             timeDurationPtr(defaultRetryPeriod),
+		RenewDeadline:           timeDurationPtr(defaultRenewDeadline),
+		LeaseDuration:           timeDurationPtr(defaultLeaseDuration),
```

Comment:
@kevinrizza I tracked down a different implementation after looking more into this offline. I'm concerned with the complexity needed to dynamically disable leader election on SNO-enabled clusters, while accounting for API server disruptions in HA clusters, and having to provide a potential escape hatch. After diving into the BZ and the linked PR, it should be fine to use the recommended leader election defaults that were provided. Any concerns with this approach? It seems like the most straight-forward implementation.

Reply:
sgtm